doc/compe.txt

```diff
@@ -37,6 +38,25 @@ CONCEPT                                                          *compe-concept*
 
 
 
+==============================================================================
+FEATURES                                                        *compe-features*
+
+- VSCode compatible expansion handling
+  - rust-analyzer's `Magic Completion`
+  - vscode-html-languageserver-bin's closing tag completion
+  - Other complex expansion are supported
+- Flexible Custom Source API
+  - The source can support `documentation` / `resolve` / `confirm`
+- Better fuzzy matching algorithm
+  - `gu` can be matched `get_user`
+  - `fmodify` can be matched `fnamemodify`
+  - See `lua/compe/matcher.lua:L57` for implementation details if you're interested
+- Buffer source carefully crafted
+  - The buffer source will index buffer words by filetype specific regular
+    expression if needed
```

Comment:
Really sorry and thank you very much.

I will be careful from now on.

Reply:
no need to sorry! :)